networkx/utils/tests/test_backends.py

```diff
@@ -74,15 +74,52 @@ def convert_to_nx(obj, *, name=None):
         assert type(nx.from_scipy_sparse_array(A)) is nx.Graph
         assert side_effects == [1]
         assert (
-            type(nx.from_scipy_sparse_array(A, backend="nx-loopback")) is LoopbackGraph
+            type(nx.from_scipy_sparse_array(A, backend="nx_loopback")) is LoopbackGraph
```

Comment:
using `isinstance` instead of `type`
```suggestion
            isinstance(nx.from_scipy_sparse_array(A, backend="nx_loopback"), LoopbackGraph)
```

Reply:
Using `isinstance` is generally good practice, but this is a test that is testing something _specific_. We have subclasses that `isinstance` would also catch. So, I prefer to be explicit here.